target/linux/ramips/image/mt76x8.mk

```diff
@@ -41,6 +41,13 @@ define Device/hc5661a
 endef
 TARGET_DEVICES += hc5661a
 
+define Device/hc5611
```

Comment:
Please change it to `define Device/hiwifi_hc5611`

Reply:
There is already hc5661 hc5661 hc5761 hc5861 hc5962 an so on, do I need to be consistent with them?
```bash
openwrt/target/linux/ramips$ grep -rin Device/hc5 image/*.mk
image/mt7620.mk:218:define Device/hc5661
image/mt7620.mk:226:define Device/hc5761
image/mt7620.mk:234:define Device/hc5861
image/mt7621.mk:86:define Device/hc5962
image/mt76x8.mk:37:define Device/hc5661a
image/mt76x8.mk:44:define Device/hc5611
```

Reply:
No, you don't need to be consistent with them.

All newly added board will use the generic board detection. Existing boards are using a target specific board detection and will be migrated to the generic one within the next month.